js/placeholders.js

```diff
@@ -0,0 +1,21 @@
+function assignPlaceholders() {     // swaps html placeholders with custom ones
+    const formList = {
+        first_name:'First name',
+        last_name:'Last name',
+        textarea_1:'Text1',
+        textarea_2:'Text2',
+        email:'email@gmail.com',
+        password:'Password',
+        vid_number:'VID',
+        tickets_count:'1'
+    };
+    for (let placeholderText in formList) {
```

Comment:
When using this kind of loop on an object, you should always add this kind of check :
```javascript
if (!formList.hasOwnProperty(placeholderText)) continue;
```
as the enumerable properties defined by Object's prototype will be used in iteration.